src/logic/safe/store/reducer/gatewayTransactions.ts

```diff
@@ -87,10 +118,14 @@ export const gatewayTransactionsReducer = handleActions<GatewayTransactionsState
         },
       }
     },
+    // If a queued transaction summary is added, it will either be added to the singular `queued.next`
+    // list or the pushed/update the `queued.queued` list by nonce in descending date order. If the
+    // number of confirmations is higher than the current stored one, the tx will be overwritten
```

Comment:
Currently, we only check that the confirmations are different, not necessarily higher.